WooCommerce/Classes/Authentication/Store Creation/Profiler/StoreCreationProfilerUploadAnswersUseCase.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import Yosemite
+
+protocol StoreCreationProfilerUploadAnswersUseCaseProtocol {
+    func storeAnswers(_ data: StoreProfilerAnswers)
+
+    func uploadAnswers() async
+}
+
+struct StoreCreationProfilerUploadAnswersUseCase: StoreCreationProfilerUploadAnswersUseCaseProtocol {
```

Comment:
Nit: It'd be great if we have some comment to explain what the use case is for.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10460/commits/5002b7510758a2bc890559ee874bdc1a2566bd47